datajoint/heading.py

```diff
@@ -191,70 +198,46 @@ def init_from_database(self, conn, database, table_name):
                     attr['dtype'] = numeric_types[(t, is_unsigned)]
         self.attributes = OrderedDict([(q['name'], Attribute(**q)) for q in attributes])
 
-    def proj(self, attribute_list, renamed_attributes, include_primary_key):
+    def project(self, attribute_list, named_attributes, force_primary_key=None):
```

Comment:
No.  This is a method of `Heading` -- It is a private method that should not be used by external users.  I renamed it because it made it easier to search.  `RelationalOperand` still only has `proj`.